packages/web/src/components/edit-playlist/desktop/EditPlaylistModal.tsx

```diff
@@ -62,8 +65,28 @@ const EditPlaylistModal = () => {
   const onCancelDelete = () => setShowDeleteConfirmation(false)
 
   const handleSubmit = useCallback(
-    (formFields: any) => {
+    (formFields: EditPlaylistValues, initialValues: EditPlaylistValues) => {
       if (playlistId) {
+        // Track any edits
+        track({
+          eventName: Name.COLLECTION_EDIT,
+          properties: {
+            id: playlistId,
+            from: initialValues,
+            to: formFields
+          }
+        })
+        // We want to pay special attention to access condition changes
+        if (initialValues.stream_conditions !== formFields.stream_conditions) {
```

Comment:
should we also track visibility changes? (e.g. hidden -> public)
We could also wait on this until the updates to editing visibility

Reply:
Yeah I would say we wait since right now its just Hidden -> Public and it doesn't really apply to collections yet anyways